pages/page_login.py

```
@@ -11,23 +11,31 @@ def __init__(self, page: Page):
+        self.page.wait_for_timeout(200)
 
-        # 버튼 비활성 예외 처리
-        for _ in range(20):
+        # 버튼 활성화 대기
+        for _ in range(40):
             if self.submit_btn.is_enabled():
                 break
-            time.sleep(0.5)
+            self.page.wait_for_timeout(300)
         else:
             raise Exception("로그인 버튼이 활성화되지 않았습니다.")
```

Ruff (0.14.8)

Create your own exception

(TRY002)